modules/proxy-certificates.adoc

```diff
@@ -0,0 +1,155 @@
+is merged with the {op-system-first} trust bundle and injected into the trust
+store of platform components that make egress HTTPS calls. For example,
+`image-registry-operator` calls an external image registry to download images.
+If `trustedCA` is not specified, only the {op-system} trust bundle is used for proxied
+HTTPS connections. Provide custom CA certificates to the {op-system} trust bundle if
+you want to use your own certificate infrastructure.
+
+The `trustedCA` field should only be consumed by a proxy validator. The
+validator is responsible for reading the certificate bundle from required key
+`ca-bundle.crt` and copying it to a ConfigMap named `trusted-ca-bundle` in the
+`openshift-config-managed` namespace. The namespace for the ConfigMap referenced
+by `trustedCA` is `openshift-config`:
+
+[source,yaml]
+----
+apiVersion: v1
+	kind: ConfigMap
+ metadata:
+	name: user-ca-bundle
+	namespace: openshift-config
+	data:
+	  ca-bundle.crt: |
+    -----BEGIN CERTIFICATE-----
+    Custom CA certificate bundle.
+    -----END CERTIFICATE-----
+----
+
+[discrete]
+== Managing proxy certificates during installation
+
+The `additionalTrustBundle` value of the installer configuration is used to
+specify any proxy-trusted CA certificates during installation. For example:
+
+----
+$ cat install-config.yaml
+. . .
+proxy:
+  httpProxy: http://<HTTP_PROXY>
+  httpsProxy: https://<HTTPS_PROXY>
+additionalTrustBundle: |
+    -----BEGIN CERTIFICATE-----
+   <MY_HTTPS_PROXY_TRUSTED_CA_CERT>
+    -----END CERTIFICATE-----
+. . .
+----
+
+[discrete]
+== Location
+
+The user-provided trust bundle is represented as a ConfigMap. The ConfigMap is
+mounted into the file system of platform components that make egress HTTPS calls.
+Typically, Operators mount the ConfigMap to
+`/etc/pki/ca-trust/extracted/pem/tls-ca-bundle.pem`, but this is not required by
+the proxy and is at the Operator's discretion.
+
+If using the RHCOS trust bundle, place CA certificates in
+`/etc/pki/ca-trust/source/anchors`.
+
+See
+link:https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/security_hardening/using-shared-system-certificates_security-hardening[Using
+shared system certificates] in the Red Hat Enterprise Linux documentation for
+more information.
+
+[discrete]
+== Expiration
+
+The user sets the expiration term of the user-provided trust bundle.
+
+The default expiration term is defined by the CA certificate itself. It is up to
+the CA administrator to configure this for the certificate before it can be used
+by {product-title} or {op-system}.
+
+[NOTE]
+====
+Red Hat does not monitor for when CAs expire. However, due to the long life of
+CAs, this is generally not an issue. However, you might need to periodically
+update the trust bundle.
+====
+
+[discrete]
+== Services
+
+When the proxy is enabled, all platform components that make egress HTTPS calls
+will use the {op-system} trust bundle. If `trustedCA` is defined, it will also be
+used.
+
```

Comment:
I find the above paragraph confusing. I suggest:
```
By default, all platform components that make egress HTTPS calls
will use the {op-system} trust bundle. If `trustedCA` is defined, it will also be
used.
```

Is Service a service running on the node (i.e. kubelet) or a Kubernetes service. I think this should be noted.